tests/integration-tests/tests/common/utils.py

```diff
@@ -90,11 +90,53 @@ def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
         sys.exit(1)
 
 
+def retrieve_pcluster_ami_without_standard_naming(region, os, version="2.8.1", architecture="x86_64"):
+    try:
+        client = boto3.client("ec2", region_name=region)
+        ami_name = f"ami-for-testing-pcluster-version-validation-without-standard-naming-{version}-{os}"
+        amis = client.describe_images(
+            Filters=[{"Name": "name", "Values": [ami_name]}, {"Name": "architecture", "Values": [architecture]}],
+        ).get("Images", [])
+        if amis:
+            # If the AMI is found, return the AMI
+            return amis[0]["ImageId"]
+        else:
+            # If no AMI is found, copy from an official AMI
+            official_ami_name = "aws-parallelcluster-{version}-{ami_name}".format(
+                version=version, ami_name=OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP.get(os).get("name")
+            )
+            official_amis = client.describe_images(
+                Filters=[
+                    {"Name": "name", "Values": [official_ami_name]},
+                    {"Name": "architecture", "Values": [architecture]},
+                ],
+                Owners=OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP.get(os).get("owners"),
+            ).get("Images", [])
+            return client.copy_image(
```

Comment:
are we going to leave these AMIs in the account forever?do we want to put in place some clean-up logic?

Reply:
Should I call deregister explicitly? Or just wait for the "cleanup_old_private_ami" on Jenkins to run?

Reply:
it would be nice to have the test properly cleanup all resources that get created without relying on external cleanup projects. What you could is to transform this into a fixture so that the cleanup is automatically performed when the test case enters the tear down phase.

https://docs.pytest.org/en/stable/fixture.html#fixture-finalization-executing-teardown-code

Reply:
Yea. Our concern was that deleting AMI in production account is 'dangerous'. Anyway, I will use fixture to remove the AMI

Reply:
that is a valid point. Maybe before triggering a deletion let's make sure the AMI we are trying to delete is a private one